cdap-app-fabric/src/test/java/io/cdap/cdap/internal/app/deploy/RemoteConfiguratorTest.java

```diff
@@ -126,7 +126,8 @@ public static void init() throws Exception {
     namespaceAdmin.create(NamespaceMeta.DEFAULT);
 
     remoteClientFactory = new RemoteClientFactory(discoveryService,
-                                                  new DefaultInternalAuthenticator(new AuthenticationTestContext()));
+                                                  new DefaultInternalAuthenticator(new AuthenticationTestContext()),
+                                                  null);
```

Comment:
shall we add a constructor with null `RemoteAuthenticator`? so that all clients dont need to be changed.

Reply:
Done